### ISSUE TITLE
Crash in io.smooch.ui.fragment.ConversationFragment

### ISSUE THREAD

**Issue by joluet** [smooch/smooch-android#89]
I observed this crash in my crash reporting tool:

```
Exception java.lang.RuntimeException: Can't create handler inside thread that has not called Looper.prepare()
android.os.Handler.<init> (Handler.java:200)
android.os.Handler.<init> (Handler.java:114)
android.widget.Toast$TN$2.<init> (Toast.java:336)
android.widget.Toast$TN.<init> (Toast.java:336)
android.widget.Toast.<init> (Toast.java:103)
android.widget.Toast.makeText (Toast.java:256)
android.widget.Toast.makeText (Toast.java:283)
io.smooch.ui.fragment.ConversationFragment$6.run ()
android.os.AsyncTask$SerialExecutor$1.run (AsyncTask.java:243)
java.util.concurrent.ThreadPoolExecutor.runWorker (ThreadPoolExecutor.java:1133)
java.util.concurrent.ThreadPoolExecutor$Worker.run (ThreadPoolExecutor.java:607)
java.lang.Thread.run (Thread.java:761)
```

**Comment by wmora** [comment]
Which version of the SDK are you using? We solved a similar crash in [v4.6.1](https://github.com/smooch/smooch-android/releases/tag/4.6.1)

**Comment by joluet** [comment]
Ok I'm using `4.6.0`. I'll try `4.6.1`, thanks

**Comment by jugarrit** [comment]
This crash is fixed in `4.7.1`, thanks for reporting!
